qemu/tests/cfg/blockdev_commit_install.cfg

```diff
@@ -50,3 +50,14 @@
         iothread_scheme ?=
         iothreads ?=
         image_iothread ?=
+    iscsi_direct:
+        enable_iscsi_cd1 = no
+        enable_iscsi_unattended = no
+        enable_iscsi_sn1 = no
+        image_raw_device_sn1 = no
+        enable_iscsi_sn2 = no
+        image_raw_device_sn2 = no
+        enable_iscsi_sn3 = no
+        image_raw_device_sn3 = no
+        enable_iscsi_sn4 = no
+        image_raw_device_sn4 = no
```

Comment:
@aliang123 Aihua, this case would use the default iscsi lun=0 specified in the cfg file. we need to let it use lun=3 instead.